src/net.cpp

```diff
@@ -190,8 +190,9 @@ CAddress GetLocalAddress(const CNetAddr *paddrPeer, ServiceFlags nLocalServices)
 static int GetnScore(const CService& addr)
 {
     LOCK(cs_mapLocalHost);
-    if (mapLocalHost.count(addr) == 0) return 0;
-    return mapLocalHost[addr].nScore;
+    const auto it = mapLocalHost.find(addr);
+    if (it == mapLocalHost.end()) return 0;
+    return it->second.nScore;
```

Comment:
```diff
-    if (it == mapLocalHost.end()) return 0;
-    return it->second.nScore;
+    return (it == mapLocalHost.end()) ? 0 : it->second.nScore
```

Reply:
Good idea, done. Though I negated the condition to have the "found" outcome first, which seems more natural to me.